AppBuilder/platform/ABModel.js

```diff
@@ -49,12 +49,14 @@ function errorPopup(error) {
 function no_socket_trigger(model, key, data) {
    // If we do not have socket updates available, then trigger an
    // update event with this data.
-   if (model.AB.Network.type() != "socket") {
-      model.AB.emit(key, {
-         objectId: model.object.id,
-         data,
-      });
-   }
+   // ## NOTE: attempting to see performance difference
+   // ## if we reduce our socket updates
+   // if (model.AB.Network.type() != "socket") {
```

Comment:
I don't understand this. Don't we still want this check?  Won't this cause duplicate updates when we use sockets?